jobs/config.json

```diff
@@ -7797,6 +7797,23 @@
       "sig-aws"
     ]
   },
+  "ci-kubernetes-e2e-kops-gce": {
+    "args": [
+      "--aws",
+      "--cluster=e2e-kops-gce.k8s.local",
```

Comment:
k8s.local means this will use "gossip DNS", which is a little easier to start with than real-DNS, but not really the recommended configuration.  Once I get this working, I'll move to real DNS.